src/Compilers/Core/Portable/DiagnosticAnalyzer/AnalyzerDriver.cs

```diff
@@ -1384,7 +1384,8 @@ private bool ShouldExecuteOperationBlockActions(AnalysisScope analysisScope, ISy
             var symbol = symbolEvent.Symbol;
             var executeSyntaxNodeActions = ShouldExecuteSyntaxNodeActions(analysisScope);
             var executeCodeBlockActions = ShouldExecuteCodeBlockActions(analysisScope, symbol);
-            var executeOperationActions = ShouldExecuteOperationActions(analysisScope);
+            // TODO: https://github.com/dotnet/roslyn/issues/9270
+            var executeOperationActions = this.analyzerExecutor.Compilation.IsIOperationFeatureEnabled() && ShouldExecuteOperationActions(analysisScope);
```

Comment:
Just realized that since we are not gonna merge this change into master, it's unnecessary to keep track the issue in the comments (and the issue itself would be useless too). I will remove them all.